templates/rotki/meta.yaml

```diff
@@ -0,0 +1,32 @@
+name: Rotki
+description: Rotki is an open source portfolio tracking, analytics, accounting, and tax reporting tool that protects your privacy.
+instructions: null
+changeLog:
+  - date: 2023-2-9
+    description: first release
+links:
+  - label: Documentation
+    url: https://github.com/rotki/rotki
+  - label: Github
+    url: https://github.com/boypt/simple-torrent
```

Comment:
Is this right? Also, there is a website.

Reply:
forgot to change that sorry

Reply:
fixed